pkg/controller/util/node/controller_utils.go

```diff
@@ -337,73 +331,6 @@ type TenantPartitionManager struct {
 	DaemonSetStore    appsv1listers.DaemonSetLister
 }
 
-func GetTenantPartitionInsecureClient(ipAddr string) (clientset.Interface, error) {
```

Comment:
Just noticed this code. Thanks for cleaning it up!

Reply:
yeah! no dead code!